src/shared/components/case-editor/case-access-utils/index.ts

```diff
@@ -63,7 +63,7 @@ export class CaseAccessUtils {
                 roleName = `${accessType}-access-${CaseAccessUtils.ADMIN_ROLE_NAME}`;
                 break;
             default:
-                roleName = `${accessType}-access-${CaseAccessUtils.LEGAL_OPERATIONS_ROLE_NAME}`;
+                roleName = `${accessType}-access-legal-ops`;
```

Comment:
Can't we change the variable name here that was being used?

Reply:
Done